pkg/hub/manager.go

```diff
@@ -119,10 +119,10 @@ func RunControllerManager(ctx context.Context, controllerContext *controllercmd.
 	)
 
 	rbacFinalizerController := rbacfinalizerdeletion.NewFinalizeController(
-		kubeInfomers.Rbac().V1().Roles(),
-		kubeInfomers.Rbac().V1().RoleBindings(),
-		kubeInfomers.Core().V1().Namespaces().Lister(),
-		clusterInformers.Cluster().V1().ManagedClusters().Lister(),
+		kubeInfomers.Rbac().V1().Roles().Lister(),
+		kubeInfomers.Rbac().V1().RoleBindings().Lister(),
+		kubeInfomers.Core().V1().Namespaces(),
```

Comment:
we seem not filter the cluster label in the  kube informer in the 2.2, can do we add the cluster label filter in the kube inform now?